evaluate.py

```diff
@@ -125,8 +127,12 @@ def pump_it(self):
 
         # TODO: validate the object
         try:
+            # Change the reader config if a source file was specified via args
+            if args.source_xml:
+                profile["pipeline"]["reader"][1]["file_path"] = args.source_xml
```

Comment:
That looks a bit dangerous as you are relying on an order of arguments here, so things can explode

Reply:
Yep, indeed it is quite brittle, but I don't see another way unless we deserialise the YAML in to first class config objects.

Mind you, I can make this a little less brittle by finding the first entry that has a `file_path` and updating only that one – will make that change.